packages/demo/src/App.svelte

```diff
@@ -602,6 +602,10 @@
   const updateTheme = () => {
     onboard.state.actions.updateTheme(selectedTheme)
   }
+
+  function isSVG(str) {
```

Comment:
is this to prevent SVG Javascript Execution?

Reply:
Its to handle both svg and and img icons. Svgs are returned with 6963 which is already handled in the W3O

Reply:
I added a check on the listening of  wallet announcements - good call!

Reply:
awesome- thanks!!